colabs/chart_expressions/chart_expressions_demo.ipynb

```diff
@@ -0,0 +1,590 @@
+            ]
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ],
+            "text/html": [
+              "Find logs at: <code>./wandb/run-20250729_200345-fjg1cqde/logs</code>"
+            ]
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "# Start a new run\n",
+        "run = wandb.init(\n",
+        "    project=\"expression-demo\",\n",
+        "    name=\"training-example\",\n",
+        "    config={\n",
+        "        \"learning_rate\": 0.001,\n",
+        "        \"batch_size\": 32,\n",
+        "        \"epochs\": 50\n",
+        "    }\n",
+        ")\n",
+        "\n",
+        "# Simulate training metrics\n",
+        "epochs = 50\n",
+        "for epoch in range(epochs):\n",
+        "    # Simulate improving accuracy\n",
+        "    accuracy = 0.5 + (0.4 * epoch / epochs) + random.uniform(-0.02, 0.02)\n",
+        "    accuracy = min(accuracy, 0.98)\n",
+        "\n",
+        "    # Simulate decreasing loss\n",
+        "    loss = 2.0 * np.exp(-epoch / 10) + random.uniform(-0.05, 0.05)\n",
+        "\n",
+        "    # Simulate validation metrics (slightly worse than training)\n",
+        "    val_accuracy = accuracy - random.uniform(0.02, 0.05)\n",
+        "    val_loss = loss + random.uniform(0.05, 0.15)\n",
+        "\n",
+        "    # Simulate learning rate decay\n",
+        "    learning_rate = 0.001 * (0.95 ** (epoch // 10))\n",
+        "\n",
+        "    # Simulate memory usage\n",
+        "    memory_used = 1000 + epoch * 20 + random.uniform(-50, 50)\n",
+        "    memory_total = 8000\n",
+        "\n",
+        "    # Log metrics\n",
+        "    wandb.log({\n",
+        "        \"epoch\": epoch,\n",
+        "        \"accuracy\": accuracy,\n",
+        "        \"loss\": loss,\n",
+        "        \"val_accuracy\": val_accuracy,\n",
+        "        \"val_loss\": val_loss,\n",
+        "        \"learning_rate\": learning_rate,\n",
+        "        \"memory_used\": memory_used,\n",
+        "        \"memory_total\": memory_total\n",
+        "    })\n",
+        "\n",
+        "    time.sleep(0.1)  # Small delay to simulate training time\n",
+        "\n",
+        "print(f\"Training complete! View your run at: {run.url}\")\n",
+        "wandb.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mJq6Oogzxy8J"
+      },
+      "source": [
+        "## How to Use Expressions in W&B\n",
+        "\n",
+        "After running the cell above, click on the run URL to open your W&B dashboard. Then:\n",
+        "\n",
+        "1. **Create a Line Plot Panel**: Click \"Add Panel\" → \"Line Plot\"\n",
+        "2. **Select a Metric**: Choose one metric (e.g., `accuracy`)\n",
```

Comment:
Hmm...when I click Add panel and then Line plot, a modal appears with multiple tabs. Should I ignore it before proceeding?